drivers/exec/driver.go

```diff
@@ -252,8 +304,8 @@ func (d *ExecDriver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *c
 		User:           cfg.User,
 		ResourceLimits: true,
 		Resources: &executor.Resources{
-			CPU:      int(cfg.Resources.LinuxResources.CPUShares),
-			MemoryMB: int(drivers.BytesToMB(cfg.Resources.LinuxResources.MemoryLimitBytes)),
+			CPU:      cfg.Resources.NomadResources.CPU,
+			MemoryMB: cfg.Resources.NomadResources.MemoryMB,
```

Comment:
Looks like LinuxResources is computed off of NomadResources, so it will be fine to use whichever representation works best for your driver. https://github.com/hashicorp/nomad/pull/4844/commits/d05d09739f9c7b8eba11fe30ba0b637db27dc86c#diff-44115f774c7a50972c7d0c97054ea0a4

Since only NomadResources is set right now, your change is best! :+1: